pkg/signals/signal.go

```diff
@@ -26,7 +26,7 @@ var onlyOneSignalHandler = make(chan struct{})
 // which is closed on one of these signals. If a second signal is caught, the program
 // is terminated with exit code 1.
 func SetupSignalHandler() (stopCh <-chan struct{}) {
-	close(onlyOneSignalHandler) // panics when called twice
+	//	close(onlyOneSignalHandler) // panics when called twice
```

Comment:
This is a k8s package. This close statement will allow only one controller to run. Since we have multiple CRD controllers across the repo, calling this will panic. So we have commented, instead of removing.